pixoo/__init__.py

```diff
@@ -430,6 +450,7 @@ def __send_buffer(self):
             'PicSpeed': 1000,
             'PicData': str(base64.b64encode(bytearray(self.__buffer)).decode())
         }))
+        #print(str(base64.b64encode(bytearray(self.__buffer)).decode()))
```

Comment:
Superfluous comment/code.

Reply:
as also in line 453 I forgot to remove the comments from the programming... i am still quite a beginner but i give my best :D